<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherWindow</class>
 <widget class="QDialog" name="WeatherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Montgomery Weather Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTemp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblWind">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wind</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCondition">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Condition</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>XXÂ°C</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>XXkm/h</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>XXXX</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
